onchain/src/tests/username_store.cairo

```diff
@@ -0,0 +1,72 @@
+use username_store::UsernameStore;
+use username_store::{IUsernameStoreDispatcher, IUsernameStoreDispatcherTrait};
+use username::UserNameClaimErrors;
+
+
+
+#[cfg(test)]
+mod tests {
+
+    // Import the deploy syscall to be able to deploy the contract.
+    use starknet::{
+        ContractAddress, get_caller_address, get_contract_address, contract_address_const
+    };
+    
+    use starknet::syscalls::deploy_syscall;
+    use starknet::{ContractAddress, contract_address_const};
+
+     // Use starknet test utils to fake the transaction context.
+    use starknet::testing::{claim_username, transfer_username};
+
+    fn USERNAME_STORE_CONTRACT() -> ContractAddress {
+    contract_address_const::<'UsernameStores'>()
+}
+
+       // Deploy the contract and return its dispatcher.
+    fn deploy() -> ContractAddress {
+        // Set up constructor arguments.
+        let contract = snf::declare('UsernameStores');
```

Comment:
I notice various errors in this test file, like this should be `UsernameStore`, for instance.

The tests here won't running unless you add the modules in `onchain/src/lib.cairo` like other modules do. So, it might have caused confusion since I don't think these tests will pass now.